cmd/login/cmd.go

```diff
@@ -111,7 +112,7 @@ func init() {
 	flags.StringVar(
 		&args.env,
```

Comment:
rather than env, can we add a `--url` flag and make `--env` an alias of it. that gives people the option of which to use. we can deprecate `--env` eventually once people are used to it.

Reply:
I agree with the consistency. However, I'll let @ciaranRoche or @robpblake or the ROSA team decide if this change should be made and if so, how and when to deprecate --env.
In any case, maybe a separate MR if we decide to go through with this?

Reply:
i suspect it will never change if we dont do it now :smile: and honestly I'd pref to just rip the bandaid off and change it to `--url` here and now. I only offered the `--url`/`--env` as synonymous flags as a compromise position. Not a deal breaker if they want to keep it as `--env`, i would just prefer getting things in sync with ocm cli now while we are here.

Reply:
Right now we do not have a dep policy, so we need to maintain backwards compatibility. 

I agree on the point if its not done now it will never be done. Given env is hidden and only intended for internal use, and we plan to make url open to end users. I would recommend for now following the pattern here - https://github.com/openshift/rosa/pull/1720 to support both `--env` and `--url` and keep url hidden until such time we want to expose this to end users

Reply:
Thanks @ciaranRoche. Added `--url` as an alternative to `--env` as per the suggested MR ref.